lib/vagrant-cloudstack/plugin.rb

```diff
@@ -24,7 +24,7 @@ class Plugin < Vagrant.plugin("2")
         Config
       end
 
-      provider(:cloudstack, parallel: true) do # Setup logging and i18n
+      provider(:cloudstack, { parallel: true, box_optional:  true}) do # Setup logging and i18n
```

Comment:
This setting at least removes the need to specify the dummy box in the first place.